cmd/deployment/kibana/resync.go

```diff
@@ -28,14 +28,31 @@ import (
 )
 
 var resyncKibanaCmd = &cobra.Command{
-	Use:     "resync <cluster id>",
+	Use:     "resync [<deployment id> | --all]",
```

Comment:
As per https://github.com/elastic/ecctl/issues/44#issuecomment-562053575
```suggestion
	Use:     "resync {<deployment id> | --all}",
```

Reply:
ah nice! I had no idea 😄